app/helpers/application_helper.rb

```diff
@@ -1,2 +1,25 @@
 module ApplicationHelper
+  module ActionView
+    module Helpers
+      class FormBuilder
+        def date_select(method, options = {}, html_options = {})
+          existing_date = @object.send(method)
+          formatted_date = existing_date.to_date.strftime("%F") if existing_date.present?
+          @template.content_tag(:div, :class => "input-group") do
+            text_field(method, :value => formatted_date, :class => "form-control datepicker", :"data-date-format" => "YYYY-MM-DD") +
+            @template.content_tag(:span, @template.content_tag(:span, "", :class => "glyphicon glyphicon-calendar") ,:class => "input-group-addon")
+          end
+        end
+
+        def datetime_select(method, options = {}, html_options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.<br>Unused method argument - `html_options`. If it's necessary, use `_` or `_html_options` as an argument name to indicate that it won't be used.